stdlib/types.pyi

```diff
@@ -687,6 +687,8 @@ if sys.version_info >= (3, 10):
     class UnionType:
         @property
         def __args__(self) -> tuple[Any, ...]: ...
+        @property
+        def __parameters__(self) -> tuple[Any, ...]: ...
```

Comment:
could this be

```suggestion
        def __parameters__(self) -> tuple[TypeVar | ParamSpec | TypeVarTuple, ...]: ...
```

I think each element of the tuple will always be an instance of one of those three classes, right?

Reply:
Probably...? To be honest, I just copied this signature from `GenericAlias` (line 657) 🤷🏻.
 
So should `GenericAlias.__parameters__` then be changed too, or do we leave this one `Any` for consistency, or ...?

Reply:
eh, I guess I can merge this as-is for now and then we could see about changing them both in a followup PR. It's probably good for them to be consistent!